doc/BUILDSERVER.md

```diff
@@ -0,0 +1,98 @@
+# Buildserver
+
+## Introduction
+Most of the build jobs on the jenkins server that handles automated testing of
+contributions have been replaced by a single
+[job](https://github.com/ElektraInitiative/libelektra/).
+
+
+## Jenkinsfile
+Jenkinsfiles describe what actions the build system should execute on what
+build slave.
+Currently Elektra uses two different files.
+
+One is for daily maintanence tasks, like cleaning up build servers
+([Jenkinsfile.daily](https://github.com/ElektraInitiative/libelektra/blob/master/scripts/jenkins/Jenkinsfile.daily))
```

Comment:
There is also a master.libelektra.org to shorten URLs and have them independent of GitHub.